packages/react-components/src/components/chart/utils/getInfo.ts

```diff
@@ -46,11 +46,11 @@ export const setXWithBounds = (size: SizeConfig, x: number) => {
   return x > upperLimit ? upperLimit : x < lowerLimit ? lowerLimit : x;
 };
 
-export const getTrendCursorHeaderTimestampText = (timestampInMs: number, previousText?: string) => {
-  const title = previousText && previousText.split('\n')[0];
+export const getTrendCursorHeaderTimestampText = (timestampInMs: number) => {
+  // const title = previousText && previousText.split('\n')[0];
```

Comment:
can we delete this?

Reply:
ah! auto merge was enable, will remove in a follow up CR.